react/PlaceDetails.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { useState } from 'react'
+import { Dropdown } from 'vtex.styleguide'
+
+const address: { [index: string]: any } = {
+  street: 'Av. Belgrano',
+  number: 2248,
+  complement: '',
+  postalCode: '2000',
+  city: 'Rosario',
+  state: 'Santa Fe',
+}
+
+const countryDescriptions: CountryDescription[] = [
+  {
+    name: 'ARG',
+    summary: [
+      [{ name: 'street' }, { delimiter: ' ', name: 'number' }],
+      [{ name: 'complement' }],
+      [{ name: 'postalCode' }],
+      [{ name: 'city' }, { delimiter: ', ', name: 'state' }],
+    ],
+  },
+  {
+    name: 'BRA',
+    summary: [
+      [
+        { name: 'street' },
+        { delimiter: ' ', name: 'number' },
+        { delimiter: ', ', name: 'complement' },
+      ],
+      [
+        { name: 'neighborhood', delimiterAfter: ' - ' },
+        { name: 'city' },
+        { delimiter: ' - ', name: 'state' },
+      ],
+      [{ name: 'postalCode' }],
+    ],
+  },
+  {
+    name: 'KOR',
+    summary: [
+      [{ name: 'street' }, { delimiter: ', ', name: 'complement' }],
+      [
+        { name: 'city' },
+        { delimiter: ', ', name: 'state' },
+        { delimiter: ' ', name: 'postalCode' },
+      ],
+    ],
+  },
+]
+
+const PlaceDetails: StorefrontFunctionComponent<PlaceDetailsProps> = () => {
+  const [option, setOption] = useState<string>('ARG')
+  const [summary, setSummary] = useState<LineComponent[][]>([])
```

Comment:
Note that you're setting `ARG` as initial value of `option` but setting the initial value of `summary` as empty. This way when I load the page I see `Argentina` in the dropdown but I only see the address once I select a different country in the dropdown and then select Argentina again.

Reply:
Great, I decided to stop using a default value so that I don't have to bother getting the summary for that default country.